internal/backend/sessions/sessionworkflows/workflow.go

```diff
@@ -238,6 +239,12 @@ func (w *sessionWorkflow) initGlobalModules() (map[string]sdktypes.Value, error)
 
 	vs := make(map[string]sdktypes.Value, len(execs))
 
+	if w.ws.cfg.OSModule {
```

Comment:
Yet another option to support. What is the reason we don't want the `os` module?

Reply:
when deploying in cloud

Reply:
IMO the right solution is the general "isolation" discussion we have. But for now OK.